ISLE/3ds/filesystem.h

```diff
@@ -0,0 +1,8 @@
+#ifndef N3DS_FILESYSTEM_H
+#define N3DS_FILESYSTEM_H
```

Comment:
not sure why this is called filesystem, since it doesn't do anything filesystem related. I'd call it config or something.